2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,2404 @@
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Biospecimen'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Get a Biospecimen
+  /data_access_committees/{data_access_committee_id}:
+    get:
+      description: Given a DataAccessCommittee ID, get the DataAccessCommittee record
+        from the metadata store.
+      operationId: get_data_access_committees_data_access_committees__data_access_committee_id__get
+      parameters:
+      - in: path
+        name: data_access_committee_id
+        required: true
+        schema:
+          title: Data Access Committee Id
+          type: string
+      - in: query
+        name: embedded
+        required: false
+        schema:
+          default: false
+          title: Embedded
+          type: boolean
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DataAccessCommittee'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Get a DataAccessCommittee
+  /data_access_policies/{data_access_policy_id}:
+    get:
+      description: Given a DataAccessPolicy ID, get the DataAccessPolicy record from
+        the metadata store.
+      operationId: get_data_access_policies_data_access_policies__data_access_policy_id__get
+      parameters:
+      - in: path
+        name: data_access_policy_id
+        required: true
+        schema:
+          title: Data Access Policy Id
+          type: string
+      - in: query
+        name: embedded
+        required: false
+        schema:
+          default: false
+          title: Embedded
+          type: boolean
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DataAccessPolicy'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Get a DataAccessPolicy
+  /datasets:
+    post:
+      description: 'Given a list of File IDs and a Data Access Policy ID, create a
+        new Dataset with
+
+        the files and and link it to the given DAP ID.'
+      operationId: create_dataset_datasets_post
+      parameters:
+      - in: query
+        name: data_access_policy_id
+        required: true
+        schema:
+          title: Data Access Policy Id
```

Comment:
Where is the data access policy created? Could we maybe embed it into the body of this request?

Reply:
Same for the datasets?